metamod/src/game_support.cpp

```diff
@@ -69,15 +69,14 @@ const game_modinfo_t g_known_games[] = {
 	{ "recbb2",             "recb.so",                  "recb.dll",                  "Resident Evil : Cold Blood"                       },
 	{ "rewolf",             nullptr,                    "gunman.dll",                "Gunman Chronicles"                                }, // have no linux binary found!
 	{ "ricochet",           "ricochet.so",              "mp.dll",                    "Ricochet"                                         },
-	{ "rockcrowbar",        "rc.so",                    "rc.dll",                    "Rocket Crowbar"                                   },
-	{ "rockcrowbar",        "rc_i386.so",               "rc.dll",                    "Rocket Crowbar"                                   },
+	{ "rockcrowbar",        "rc.so",                    "rc.dll",                    "Rocket Crowbar"                                   }, // + _i386.so
 	{ "rspecies",           "hl.so",                    "hl.dll",                    "Rival Species"                                    },
 	{ "scihunt",            "shunt.so",                 "shunt.dll",                 "Scientist Hunt"                                   },
 	{ "ship",               "ship.so",                  "ship.dll",                  "The Ship"                                         },
 	{ "si",                 "si.so",                    "si.dll",                    "Science & Industry"                               },
 	{ "snow",               "snow.so",                  "snow.dll",                  "Snow-War"                                         },
 	{ "stargatetc",         "hl.so",                    "hl.dll",                    "StargateTC (Old, 1.x)"                            },
-	{ "stargatetc",         "stc",                      "hl.dll",                    "StargateTC (Updated, 2.x)"                        }, // _i386.so
+	{ "stargatetc",         "stc.so",                   "hl.dll",                    "StargateTC (Updated, 2.x)"                        }, // _i386.so
 	{ "stargatetc",         "stc_i386_opt.so",          "hl.dll",                    "StargateTC (Updated, 2.x)"                        }, // ...
```

Comment:
but i still not shure about this file. 

"stc_i386_opt.so" -> "stc_opt.so" ??

Reply:
no, it should remain `stc_i386_opt.so`